packages/cosmic-swingset/package.json

```diff
@@ -23,7 +23,15 @@
     "@agoric/evaluate": "^2.2.0",
     "@agoric/eventual-send": "^0.5.0",
     "@agoric/harden": "0.0.4",
```

Comment:
Does cosmic-swingset depend directly on these? If they're transitive dependencies, we don't really need to name them from cosmic-swingset.

Reply:
Hmm, what's the difference? If `lib/ag-solo/` has these dependencies (which it does), which category does that fall in?

Reply:
That sounds like it qualifies. I'd say: if any code in `package-X/src/*` says `import Y`, then X depends on Y. `lib/` counts as source (`test/` would make it a `devDependency`).

(there were so many dependencies added that I thought maybe we were adding all the new packages just to be conservative; also at some point in the past, the transitive-crawler wasn't working right and we had to include the whole tree as a workaround, but I think that should be fixed now)

Reply:
Ok, great!